### PR TITLE
jeremy my first string and list exercise

### PR DESCRIPTION
**Pull request by jtbeeker**
My first string and list exercise gave me a headache...

http://jtbeeker.github.io/spring2014/2014/01/31/jeremy_stringexercise.html

### CODE REVIEW

File: _posts/2014-01-31-jeremy_stringexercise.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+```
+
+even sum in 2 3 6 9 2
+
+```
+
+## list ex 9
+
+Input:
+```
```
Comment (eah13) [inline _posts/2014-01-31-jeremy_stringexercise.md:181]: You need a blank line above 180 here.

**Comment by eah13** [comment]
@jtbeeker sorry you got a headache.  An account of where an why in your post might be nice.

Your formatting is off towards the bottom of the post.  Fix that and you're ready for merge.

**Comment by jtbeeker** [comment]
i think i forgot to space after "output"?

**Comment by gerbal** [comment]
@jtbeeker It looks like that fixed it. Comment blocks need to have an empty line before the ```

**Comment by gerbal** [comment]
looks good, merging

**Pull request merged**